src/main/java/gov/nasa/jpl/mbee/mdk/mms/sync/delta/DeltaSyncRunner.java

```diff
@@ -319,6 +319,8 @@ else if (shouldUpdate && mmsChange != null) {
             }
             if (!postElements.isEmpty()) {
                 URIBuilder requestUri = MMSUtils.getServiceProjectsRefsElementsUri(project);
+                // Prevent potential "Not Equivalent" errors due to default merging of element data.
+                requestUri.addParameter("overwrite", "true");
```

Comment:
there's a danger that if the element being sent is a view, adding overwrite can remove the generated view information since those are only sent by view generation

Reply:
Nice catch @dlamoris. I can't think of a simple way to address that important case short of separating views to be committed into their own request without `overwrite`. Implementing and sufficiently testing that will likely take a while and I don't believe we want to block the release for it, so we may want to revert the changes for and re-open the issue to be addressed in a future release. Thoughts?

Reply:
yea, sounds good
I think the other change is ok since it's a manual action right? Should probably highlight if that action is done on a view it'll have the same issue